themes/tiddlywiki/vanilla/sidebar-resizer.tid

```diff
@@ -0,0 +1,158 @@
+title: $:/themes/tiddlywiki/vanilla/sidebar-resizer
+tags: [[$:/tags/Stylesheet]]
+code-body: yes
+
+\define sidebarbreakpoint()
+<$text text={{$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint}}/>
+\end
+
+\define sidebarbreakpoint-minus-one()
+<$text text={{{ [{$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint}removesuffix[px]subtract[1]addsuffix[px]] ~[{$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint}] }}}/>
+\end
```

Comment:
Both of these macros could be implemented as functions. We should not be introducing new macros to the core.

Reply:
Yes, I just copied them 1:1 from the base stylesheet. Maybe we should move them to another place like `$:/core/wiki/functions/CSS` so that we can reuse them?

Reply:
Done